app/views/users/_form.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_for(@user) do |f|
+  %p First Name:
+  = f.text_field :first_name, :class => "form-control", required: true, autocomplete: "off"
```

Comment:
I'd suggest you just use ruby 1.9 syntax for your hash declarations and don't mix things up too much.
```
class: "form-control"
```